### ISSUE TITLE
Can we get a product update? Or some kind of roadmap? Or slack invite?

### ISSUE THREAD

**Issue by mkpt** [voltraco/beta#87]
Hi Voltra,

I have a few issues related to communications.

- I signed up for a slack invite to monitor the progress of the work--no invite received. 
- I signed up for the Android beta--no notification or timeline/status received indicating when that might come out. 
- I post issues here, but get no response to my questions beyond a "thumbs up" which tells me you see it, but doesn't really tell me anything as a customer EX [1](https://github.com/voltraco/beta/issues/82) and [2](https://github.com/voltraco/beta/issues/84). 
- Your last tweet was from [Feb 23](https://twitter.com/BrkmstrCylinder/status/967208607745040386). Quiet social media generally means bad news.
- The [last blog entry](https://blog.voltra.co) on the site is from Jan 4, so it's been almost 3 months.
- The [last update to the Desktop app](https://docs.voltra.co/desktop/release.html) was two months ago.

All this begs the questions:

1. Is Voltra dead?
2. Is the desktop app still being worked on? When can we expect another desktop release? 
3. What's the status of the Android app beta? When can we expect to hear about beta testing?
4. Is there someone monitoring the Slack invites? Do you send out regular status emails? You might be communicating this, and I could be missing it.

It looks like a small team, but as a customer I'd at least like to know a little of what to expect in the next month, or even the next quarter at this point. I don't want to go back to Google All Access. I think you guys have a lot of potential and a great design. Please let me know.

**Comment by heapwolf** [comment]
> I signed up for a slack 

We're not doing slack any more, the majority of people seem to prefer email.

> I signed up for the Android beta

We haven't announced an ETA for this. But, it's a low priority at the moment. Is that specifically what you're waiting for?

> I post issues here, but get no response to my questions beyond a "thumbs up"

This is nuanced. But sometimes a thumbs up about all we can say. We don't have any timeframe for adding features or resolving issues.

> Your last tweet was from Feb 23.
> The last blog entry on the site is from Jan 4

True. People don't interact with us much, it's nice to hear that someone is listening :)

> The last update to the Desktop app was two months ago

That's not totally unusual. Releases in the past have been anywhere between 1-3 months, it just depends on the complexity of the release.

> Is Voltra dead?

No. It's moving at about the same pace it always has.

> Is the desktop app still being worked on

Yes. Same as above.

> When can we expect another desktop release?

April 15-ish

> What's the status of the Android app beta

In progress. The entire mobile app is being rewritten in React Native.

> When can we expect to hear about beta testing?

We don't have an ETA. And this seems like a really important factor for you. If you like Voltra, you can support the project. If you are literally waiting on the android app and have also decided to pay while you wait we can happily/easily refund you.

> Is there someone monitoring the Slack invites?

We moved to sending out status updates to email. Check your spam or that you haven't unsubscribed?

> It looks like a small team, but as a customer I'd at least like to know a little of what to expect in the next month.

We've been super focused on building out the store, so there hasn't been much time for anything else. There are some non-technical challenges there that we have been trying to work though. We also have been working on handling shipping complexity for the VMS orders. Handling logistics on shipping over a certain quantity can get more complex than we realized.

> I think you guys have a lot of potential and a great design

Thanks! Voltra is quite ambitious. Did you know Voltra is not a startup? We don't have VC. I'll explain why...

Did you know only 27% of Spotify's users pay for their service? They lost roughly 200 million dollars last year. Pandora lost about 61 million dollars last year. Tidal is basically out of money. 47% of all listens to music on YouTube for free. These are stats from the annual IFPI report (2017). My point is, this is an industry where very few people actually pay for services. Google is shutting down Play and merging it with their new version of YouTube Red (which is basically shutting down). Even Bandcamp (who is arguably one of the only companies making a profit in this industry) is pivoting to focus on music merchandise.

Voltra is a niche service. One for people who want to *own* music and support artists. VC's generally don't fund a niche services like Voltra. A VC's objective isn't to make the world better — that's what a philanthropist does. A VC plainly wants to multiply their capital (and usually as fast as possible). We knew this to start with but didn't care, we just wanted to make Voltra. 

Now, if the "big players" can't make a profit, how is Voltra going to stay in business, especially without venture capital? Clearly, we don't do it for the money. We never have. We're essentially patrons of the arts and since we care about music so much we work on this software. Aside from contributions from wonderful people like yourself we are limited in resources and time. Anyone who is contributing financially to Voltra but isn't happy with the progress or feels like they aren't getting what they expected, we're happy to refund quickly, easily and without any hard feelings. ❤️

**Comment by mkpt** [comment]
Thanks so much for the update! I appreciate the transparency. 

>We haven't announced an ETA for this. But, it's a low priority at the moment. Is that specifically what you're waiting for?

Yes. The transition from desktop-to-mobile is something that literally only Google seems to get right. I can't find another company that offers wi-fi sync of playlists between desktop and mobile without a lot of hassle. 

>We don't have an ETA. And this seems like a really important factor for you. If you like Voltra, you can support the project. If you are literally waiting on the android app and have also decided to pay while you wait we can happily/easily refund you.

I honestly was just hoping for a guestimate, e.g. "We are targeting Q2 2018 for Android beta testing." If the app is being redesigned and the focus at the moment is on the store, that helps set expectations.

Thanks so much!
